scipy/signal/_fir_filter_design.py

```diff
@@ -834,6 +835,299 @@ def remez(numtaps, bands, desired, *, weight=None, type='bandpass',
                             maxiter, grid_density)
 
 
+def _oddround(x):
+    """Return the nearest odd integer from x."""
+
+    return x - np.mod(x, 2) + 1
+
+
+def _oddceil(x):
+    """Return the smallest odd integer not less than x."""
+
+    return _oddround(x + 1)
```

Comment:
A matter of personal taste: The functions `_oddround` and `_oddceil` are each only used one time in the following code. In-lining these functions (inlcuding a comment) would make it a tiny bit easier to reason about the code.